src/components/ContentPreview/ContentPreview.js

```diff
@@ -43,38 +43,39 @@ import '../base.scss';
 import './ContentPreview.scss';
 
 type Props = {
-    fileId?: string,
-    previewLibraryVersion: string,
-    isLarge: boolean,
-    autoFocus: boolean,
-    useHotkeys: boolean,
-    contentSidebarProps: ContentSidebarProps,
-    contentOpenWithProps: ContentOpenWithProps,
-    canDownload?: boolean,
-    hasHeader?: boolean,
     apiHost: string,
     appHost: string,
-    staticHost: string,
-    staticPath: string,
-    token: Token,
-    className: string,
-    measureRef: Function,
-    onLoad: Function,
-    onNavigate: Function,
-    onDownload: Function,
-    onClose?: Function,
-    language: string,
-    messages?: StringMap,
+    autoFocus: boolean,
     cache?: APICache,
+    canAnnotate?: boolean,
```

Comment:
Does this prop reflect permissions or is it a configuration flag?

Reply:
just config, pass the option through to box content preview and eventually box annotations. I believe box annotations is where the permissions are checked